shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -410,6 +410,8 @@ bool TextInputChannel::FilterEvent(Ecore_Event_Key* event) {
 #ifdef WEARABLE_PROFILE
   // Hardware keyboard not supported on watches.
   bool is_ime = true;
+#elif __X64_SHELL__
+  bool is_ime = false;
```

Comment:
Shouldn't the else statement below be used?

Reply:
Unfortunately, I can get ``ecore_device_name_get`` at PC